src/components/ProductTour/ProductTour.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import styles from './ProductTour.scss';
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+    /** Should the tour be shown */
+    isOpen: boolean;
+    /** An array of ReactElement where each element is one slide */
+    children: React.ReactElement[];
+    /** The label for the checkbox. If undefined or empty, checkbox will not be shown */
+    checkboxLabel?: string;
+    /** Cancel button label - required if there are is then 1 slide */
+    cancelLabel?: string;
+    /** Continue button label - required if there are is then 1 slide */
+    continueLabel?: string;
+    /** Finish button label */
+    finishLabel: string;
+    /** A callback if the user clicked the Cancel button without finishing the tour */
+    onCancel?: (isChecked?: boolean) => void;
+    /** A callback if the user clicked the Finish button */
+    onFinished: (isChecked?: boolean) => void;
+}
```

Comment:
I'm wondering how I can go one step back if I by mistake to push the "continue" button? @carlobernardini

Reply:
There are dots in the footer that each represent a slide. You can click on any of these dots to move back and forth to any slide you wish.

<img width="936" alt="Screenshot 2021-02-23 at 08 11 01" src="https://user-images.githubusercontent.com/798175/108812072-bd8fa880-75ae-11eb-99e2-9a2399821fc0.png">

Reply:
So these dots should be clickable. It makes sense. According to the story, they only indicate a step, and navigation should be done on the right side of the bottom.

Reply:
Yes, I forgot to describe this functionality. My mistake.

Reply:
Speaking of which... do we need additional styling on hover (apart from `cursor: pointer`)?

Reply:
No, it's fine as is (except indeed for the pointer)